lib/matplotlib/axes/_axes.py

```diff
@@ -632,22 +632,21 @@ def annotate(self, *args, **kwargs):
         s : string
             label
 
-        xy : (x, y)
-            position of element to annotate
+        xy : (x, y) , default: "data"
```

Comment:
Oh sorry I'll remove the default value for `xy`. Yes what you said makes it much clearer but then I think I'll have to use the next line. Is it fine if I do so? Ideally I don't really want to do that though...